transformation/src/test/java/org/opensearch/migrations/transformation/rules/IndexMappingTypeRemovalTest.java

```diff
@@ -318,4 +320,75 @@ void testApplyTransformation_multiTypeUnion_withConflicts() {
         var exception = assertThrows(IllegalArgumentException.class, () -> applyTransformation(indexJson, transformer));
         assertThat(exception.getMessage(), containsString("Conflicting definitions for property during union age"));
     }
+
+    @Test
+    @SneakyThrows
+    void testApplyTransformation_multiTypeUnion_withConflicts_es5Format() {
```

Comment:
confirming that the conflict here is that types 1 and 2 both have a field1, but it's a reconcilable conflict because they're both `text` with the same settings?

Reply:
Yup!